inscriptions/ISic001094.xml

```diff
@@ -192,54 +188,30 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140578">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AR">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — — — — — — — — — ὑπὲρ σωτηρίας</supplied>
-                    <lb n="2"/>
-                    <supplied reason="lost">Αὐτοκράτορος</supplied>
-                    Καίσαρ
-                    <supplied reason="omitted">ο
-                    </supplied>
-                    <supplied reason="lost">ς — — — — —</supplied>
-                    <lb n="3"/>
-                    <supplied reason="lost">— — — — — — Α</supplied>ὐτοκρά
-                    <supplied reason="lost">τορος — — —
-                        —
-                    </supplied>
-                    <lb n="4"/>
-                    <supplied reason="lost">— — — — — — —</supplied>ου Σεβα
-                    <supplied reason="lost">στοῦ υἱοῦ — —
-                    </supplied>
-                    <lb n="5"/>
-                    <supplied reason="lost">— — — — — —
-                        —
-                    </supplied>
-                    Σεπτί
-                    <supplied reason="omitted">μ</supplied>
-                    <supplied reason="lost">ιος — —
-                        ἀνέθη</supplied>-
-                    <lb n="6"/>
-                    <supplied reason="lost">κεν τὴν παλαί</supplied>στραν
-                    κα
-                    <supplied reason="lost">ὶ τὰ ἐν αὐτῇ</supplied>
-                    <lb n="7"/>
-                    <supplied reason="lost">καθιερώσαντος
-                    </supplied>
-                    <unclear>Τ</unclear>. Φ<supplied reason="omitted">λ</supplied>αουίο
-                    <supplied reason="lost">υ — —</supplied>
-                    <lb n="8"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/> καισαρ<gap reason="illegible" quantity="1" unit="character"/> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost" cert="medium">α</supplied>υτοκρα <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/> ου σεβα <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="4"/><gap reason="lost" extent="unknown" unit="character"/> σεπτικ <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="5"/><gap reason="lost" extent="unknown" unit="character"/> στραν κα <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
`<gap reason="lost" extent="unknown" unit="character"/>στραν κα<gap reason="lost" extent="unknown" unit="character"/>`
i.e. I would close up the gaps, since you think you know there's a word break in the middle, but not before / after these letter groups.